### ISSUE TITLE
Investigate Mobile Commons Timestamp Inconsistences 

### ISSUE THREAD

**Issue by jessleenyc** [DoSomething/quasar#158]
_BUG_
-------------------
#### Current Behavior
Some mobile commons messages are showing inaccurate timestamps in our database. See the below screenshot for examples of messages activated at 1969-12-31 or 2048-12-03. 

#### Desired Behavior
Timestamps should reflect the actual time a message was received/sent. 

#### Why This Matters
We noticed this bug when we saw 'null' mobile signups on looker dashboards. The reason is twofold -- 1) we haven't fully backfilled old mobile commons campaigns with campaign run ids 2) it appears that there are new signups for these old campaigns because the timestamp was off when originally activated. 

Overall, our mobile signup count is off if timestamps are not accurate. We are not seeing this display in the mobile commons UI. 

#### Relevant Screenshots + Links
Screenshot located here - https://dosomething.slack.com/files/nick/F1S56L8E4/screen_shot_2016-07-15_at_11.09.29_am.png

**Comment by sheyd** [comment]
@jessleenyc Can you attach/embed actual screenshots?

**Comment by sheyd** [comment]
@jessleenyc @nmccormick619 So, verified that the numbers aren't wrong coming from Mobile Commons API (at least from the random sampling of phone numbers I pulled from the screenshot source numbers). I've also looked at the ETL script that updates the table, and I don't see an obvious way it would have had such wide variance of values. 

Before I spend a bunch more time investigating this, can I get some scope on the following:
1. How much of our user data and dashboards are impacted by this issue.
2. How much time should I spend on this, given that it's Data 1.0 and the `mobile_subscriptions` table, which is the last table we want to touch since it's the most esoteric.
